server/controllers/game.ts

```diff
@@ -74,6 +74,23 @@ gameController.get({ path: '/play', userType: UserType.TEACHER }, async (req: Re
   res.sendJSON(game);
 });
 
+//--- Get number of games available ---
+gameController.get({ path: '/isGameCreateBeforePlaying', userType: UserType.TEACHER }, async (req: Request, res: Response, next: NextFunction) => {
+  if (!req.user) {
+    next();
+    return;
+  }
+  const userId = req.user.id;
+  const villageId = req.user.type >= UserType.TEACHER ? parseInt(getQueryString(req.query.villageId) || '0', 10) || null : req.user.villageId;
+  if (!villageId) {
+    next();
+    return;
+  }
+  const game = await getRepository(Game).createQueryBuilder('game').where('`game`.`userId` = :userId', { userId: userId }).getCount();
+  res.sendJSON({
+    game: game,
```

Comment:
Same here, how the frontend can tell this is a number and not a game?

**Suggestion:** use `gameCount`